frontend/src/store/index.ts

```diff
@@ -6,12 +6,17 @@ import Vue from "vue";
 import Vuex from "vuex";
 
 import user from "./modules/user";
+import { Ability } from "@casl/ability";
 
 Vue.use(Vuex);
 
 export const store = new Vuex.Store({
   state: {},
-  getters: {},
+  getters: {
+    ability() {
+      return new Ability([]);
+    }
+  },
```

Comment:
What is this for?

Reply:
?

Reply:
Since the ability rules were updated in the store I thought it would be better to put in the vuex store but this can be moved into it's own file and imported where rules are updated.

Reply:
Maybe this should be a part of the `user` store module?